knimin/lib/tests/test_data_access.py

```diff
@@ -297,6 +297,99 @@ def test_get_ag_barcode_details(self):
             del(obs[key]['registered_on'])
         self.assertEqual(obs, exp)
 
+    def test_plate_operations(self):
```

Comment:
I see. If I put them independently, these functions will call each other in order to complete a test (e.g., create_plate -> set_plate_info -> get_plate_info -> delete_plate). Will this be a problem?